src/build/optimize-streams.ts

```diff
@@ -60,7 +60,7 @@ export class GenericOptimizeStream extends Transform {
       } catch (error) {
         logger.warn(
             `Unable to optimize ${this.validExtension} file ${file.path}`,
-            {err: error});
+            {err: error.message || error});
```

Comment:
The second one :) we expect the error to be from a parse error in the file, and not in the tool itself. So the message the tool gives us will be more important to the user, who can use it to find and fix their error.